webapp/brilla-frontend/components/footer.tsx

```diff
@@ -0,0 +1,46 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTwitter, faFacebook, faInstagram } from '@fortawesome/free-brands-svg-icons';
```

Comment:
we currently don't have any social pages so let's remove them for now though it's in the mockup. Once we do have we can add them so we don't have unnecessary code lingering around